src/components/UserSelector.tsx

```diff
@@ -1,33 +1,98 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { User } from '../types/User';
+
+type Props = {
+  setSelectedUser: (user: User) => void;
+  userSelect: User[];
+  selectedUser: User | null;
+  fetchPosts: (id: number) => void;
+  setIsPostsLoading: (value: boolean) => void;
+  setPostIdComments: (val: null) => void;
+};
+
+export const UserSelector: React.FC<Props> = ({
+  setSelectedUser,
+  userSelect,
+  selectedUser,
+  fetchPosts,
+  setIsPostsLoading,
+  setPostIdComments,
+}) => {
+  const [isSelectorActive, setIsSelectorActive] = useState(false);
+
+  // const handleSelectorBlur = () => {
```

Comment:
Don't forget to delete the comments

Reply:
I have a bug with users from server select blur, to replicate it just open user select and try to blur/unfocus appeared user list. Its impossible and I dont know how to give focus to available user list